experimenter/experimenter/experiments/admin.py

```diff
@@ -13,7 +13,7 @@
     NimbusBranchChangeLogSerializer,
```

Comment:
we don't need to change here

Reply:
@yashikakhurana means no need to modify the changes in this file am i right ?

Reply:
yes

Reply:
@yashikakhurana once check out i have the changes in that file